lecture_1/homework/app/controllers/api/v1/clans_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    class ClansController < ApplicationController
+      def index
+        clans = Clan.all
+
+        render json: clans.to_json(only: %i[id name])
+      end
+
+      def create
+        clan = Clan.create!(clan_params)
```

Comment:
chyba chodziło Ci tutaj o `new`, `create!` już od razu tworzy obiekt, co następny `save` pozbawia trochę sensu i przez to że to wersja z wykrzyknikiem, to będzie na nas krzyczeć errorami, przez co nawet do `save = false` nie dojdzie :D

Reply:
Racja! Poprawiłem, dzięki :-)